tests/integration/network.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package integration
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/tests/workload"
+)
+
+var (
+	ErrUnableToConfirmTx = errors.New("unable to confirm transaction")
+	ErrInvalidURI        = errors.New("invalid uri")
+	ErrTxNotFound        = errors.New("tx not found")
+)
+
+const (
+	txCheckInterval = 100 * time.Millisecond
+)
+
+type Network struct {
+	uris []string
+}
+
+func (*Network) Nodes() (out []workload.TestNode) {
+	for _, instance := range instances {
+		out = append(out, instance)
+	}
+	return out
+}
+
+func (i *instance) SubmitTxs(ctx context.Context, txs []*chain.Transaction) error {
+	errs := i.vm.Submit(ctx, true, txs)
+	if len(errs) == 0 {
+		return nil
+	}
+	return errs[0]
+}
```

Comment:
This will only issue the txs to an individual node, whereas in the e2e environment if we issue a tx then it will be immediately gossiped to the rest of the network. We should not expose this detail to test writers.

Reply:
done